OnShift/SendBird-JavaScript
# PL 59 hide tooltips mobile

# Things Done
- hide tooltip on mobile views
  - defaults to 'none' display
- add way to start demo from `package.json`
  - foolishly removed

# How To Test
- open widget in both mobile and desktop view
- in desktop view, you should be able to see the tooltips 
- in mobile, no such luck
